.github/workflows/test.yml

```diff
@@ -140,6 +140,7 @@ env:
   AUDIT_API_USERNAME: ${{ secrets.AUDIT_API_USERNAME }}
   AUDIT_API_PASSWORD: ${{ secrets.AUDIT_API_PASSWORD }}
   UTILISATION_REPORT_CREATION_FOR_BANKS_SCHEDULE: ${{ vars.UTILISATION_REPORT_CREATION_FOR_BANKS_SCHEDULE }}
+  UTILISATION_REPORT_CREATION_FAILURE_EMAIL_ADDRESS: ${{ secrets. UTILISATION_REPORT_CREATION_FAILURE_EMAIL_ADDRESS }}
```

Comment:
Does this need to be a secret or can it just be a variable?

Reply:
CONTACT_US_EMAIL_ADDRESS is stored as a secret so thought should keep this the same

Reply:
absolutely fair for this, I'd argue contact us email is pretty public (accessible by anyone using the system) and so shouldn't be a secret but happy to leave this as a secret because it's minor anyway - it's just slightly easier to manage in the future as a variable. Will resolve this comment :)